changelog.d/6062.bugfix

```diff
@@ -0,0 +1 @@
+Use unstable prefix for 3PID unbind API.
```

Comment:
rather than have two separate changelog entries for this, I suggest updating the existing changelog entry (since it mentions r0), and making this one match (so that both PRs are linked under the same entry when we generate the changelog doc).

Reply:
Okay, updated!